CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to Young N Ambitious
+
+Thank you for your interest in contributing to Young N Ambitious (YNA)! This
+document is a set of guidelines to help you contribute to this project.
+
+## Code of Conduct
+
+By participating in this project, you are expected to uphold our
+[Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## Project Documentation
+
+Please consult the [README](./README.md) at the root of this repository.
+
+## How to Contribute
+
+Before you start writing code, make sure there is an issue open. Pull requests
+without a link to an existing issue won't be merged.
+
+If you want to get started contributing code to this project but don't know
+exactly what to work on, we compiled a good list of issues labeled as
+['good first issues'](https://github.com/Young-N-Ambitious/yna_backend/labels/good%20first%20issue)
+which are small in scope and not so complex to solve. There are also issues
+labeled as
+['help wanted'](https://github.com/Young-N-Ambitious/yna_backend/labels/help%20wanted)
+which can be a bit more complex but are good examples of things we are currently
+accepting help from the community. Lastly if you are participating in
+Hacktoberfest, look for issues labeled
+['hacktoberfest'](https://github.com/Young-N-Ambitious/yna_backend/labels/hacktoberfest).
+
+# How To Contribute - Set Up
+
+1. First you make a fork of the repo so that you can use your fork to update the
+   project.
+   1. After forking the repo in your local terminal:
+      1. create a YNA directory by running `mkdir YNA`
+      2. cd into the YNA directory by running `cd YNA`
+      3. Clone the project to your local terminal by running git clone
+         `git@github.com:{YOUR_GITHUB_USERNAME}/yna_backend.git`
+      4. cd into the YNA backend directory by running `cd yna_backend`
+      5. Now run `npm install` to install all of the project dependencies.
+   2. Now that you are inside the backend directory, you need to create a
+      branch.
+      1. To create a branch run: git checkout -b
+         FD-[#<issue_number>]<commit_type>-<description/name>
```

Comment:
the branch syntax for backend should be BA-[#<issue_number>-<description/name>
**Example branch name: BA-001-enable-cors**